src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -92,6 +92,7 @@ public function import(mixed $resource, ?string $type = null, bool|string $ignor
             }
         } finally {
             --$this->importing;
+            $this->loadExtensionConfigs();
```

Comment:
why do we need this here (even in case of exception) ?

Reply:
because there is a return statement inside the try block, so there is no other chance to load the collected ext config after `--$this->importing;` instruction

Reply:
btw, it's not about the exception at all, but if it happens I guess it doesn't matter if we try to prepend some configs afterward